### PR TITLE
updating npm package and client so that it connects with `doc-inspector`

### PR DESCRIPTION
**Pull request by mdwiltfong**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@gadget-client/doc-inspector": "^1.382.0",
     "@gadget-client/my-ai-app-mdw": "^1.34.0",
```
Comment (Jamesllllllllll) [inline package.json]: Right here!

**Review by Jamesllllllllll — COMMENTED** [review]
Looks good! Do we still need the api for `my-ai-app-mdw`?

**Comment by mdwiltfong** [comment]
Nah that was a test gadget I made

**Review by Jamesllllllllll — APPROVED** [review]
Looks good!

File: package.json (hunk 1 above)
Comment (mdwiltfong) [reply package.json]: Ah I just saw this. For some reason this comment didn't appear on mobile as expected 😓 . Maybe we'll remove it in another commit.

Comment (Jamesllllllllll) [reply package.json]: No worries! I removed it and made a commit before merging

**Pull request merged**